python/docs/source/index.rst

```diff
@@ -1,49 +1,67 @@
-Vowpal Wabbit Python Wrapper
-============================
+Vowpal Wabbit Python Wrapper Documentation
+==========================================
 
-Vowpal Wabbit is a fast machine learning library for online learning, and this is the python wrapper for the project.
+The Vowpal Wabbit (VW) project is a fast out-of-core learning system sponsored by Microsoft Research and (previously) Yahoo! Research.
```

Comment:
Why change the wording here?

Reply:
Initially, I was thinking to add general details of vw at the index page but It is not helping. I will remove it